Wrappers/Python/cil/plugins/ccpi_regularisation/functions/regularisers.py

```diff
@@ -350,9 +351,9 @@ def __rmul__(self, scalar):
         # f = alpha * f
 
     def check_input(self, input):
-        if len(input.dimension_labels) == 2:
+        if len(input.shape) == 2:
```

Comment:
I though CIL and SIRF data containers and CIL geometries had a `ndim` attribute, but I might be wrong.

Reply:
SIRF's `DataContainer`s don't have `ndim` defined but have `shape`, https://github.com/SyneRBI/SIRF/blob/1bb1b111a80f48df5e491a8ee9201715d4e9db42/src/common/SIRF.py#L47